modules/luci-mod-status/htdocs/luci-static/resources/view/status/include/00_pstore.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+'require baseclass';
+'require rpc';
+'require ui';
+
+var callFileList = rpc.declare({
+	object: 'file',
+	method: 'list',
+	params: [ 'path' ],
+	expect: { entries: [] },
+	filter: function(list, params) {
+		var rv = [];
+		for (var i=0; i<list.length; i++)
+			if (! list[i].name.match(/^crashcount/))
+				rv.push(params.path + list[i].name);
+		return rv.sort();
+	}
+});
+
+var callFileRead = rpc.declare({
+	object: 'file',
+	method: 'read',
+	params: [ 'path' ],
+	expect: { data: '' },
+	filter: function(value) {
+		return value.trim()
+	}
+});
+
+return baseclass.extend({
+	title: _('Warning'),
+
+	load: function() {
+		return Promise.all([
+			L.resolveDefault(callFileList('/sys/fs/pstore/'), {}),
+			[]
+		]).then(function(data) {
+			if (data[0].length==0)
+				return [ data[0] ];
+			for (var i=0; i<data[0].length; i++)
+				data[1][i] = L.resolveDefault(callFileRead(data[0][i],false), '');
+			return Promise.all([ data[0], Promise.all(data[1]) ]);
+		});
+	},
+
+	render: function(result) {
+		if (result[0].length<=0)
+			return;
+		var entries=result[0];
+		var content=result[1];
+
+		var filetabs = E('div', {}, E('div'));
+		for (var i=0; i<entries.length; i++) {
+			filetabs.firstElementChild.appendChild(E('div', { 'data-tab': entries[i], 'data-tab-title': entries[i] }, [
+				E('textarea', {
+					'id': 'pstore',
```

Comment:
Should the id not be uniq

Reply:
Not sure, i think within the div "data-tab" it is unique

Reply:
?